erpnext/crm/doctype/lead/lead.js

```diff
@@ -49,6 +49,16 @@ erpnext.LeadController = frappe.ui.form.Controller.extend({
 		}
 	},
 
+	validate: function() {
+			if(this.frm.doc.website) {
+				var website_url = this.frm.doc.website;
+				var regex = new RegExp('^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$');
+				if (!regex.test(website_url)) {
+						frappe.throw(__('Please enter a valid Website URL'))
+				}
+			}
+	},
```

Comment:
Add a URL control in the framework directly

Reply:
refer https://github.com/Bloomstack/frappe/pull/497